server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -387,6 +393,12 @@ public void assertContentLength(int size) throws IOException {
         assertThat(response.getFirstHeader("Content-Length").getValue()).isEqualTo(String.valueOf(size));
     }
 
+    @Then("^user \"([^\"]*)\" request the attachment \"([^\"]*)\" is still there$")
+    public void assertAttachment(String username, String attachmentId) throws Exception {
```

Comment:
I don't care about these lower level details. What I care here is:

 - Can user A downloads user B attacments/messages ? The answer is no.

I don't care about the "how" what is an implementation detail, and SHOULD be abstracted from your tests